comms/storage/monitor/monitor.go

```diff
@@ -0,0 +1,75 @@
+// Package monitor provides a way to sync up each node's state to NATS so that any node can query the state of any other node.
```

Comment:
Weather map uses this to query one node and get a list of each node's view of the hash ring